src/notification/notification.stories.ts

```diff
@@ -1,12 +1,70 @@
 import { storiesOf, moduleMetadata } from "@storybook/angular";
 import { action } from "@storybook/addon-actions";
-import { withKnobs, boolean, object } from "@storybook/addon-knobs/angular";
+import { withKnobs, boolean } from "@storybook/addon-knobs/angular";
 
-import { Component } from "@angular/core";
+import { Component, Input, EventEmitter, Output } from "@angular/core";
 
 import { NotificationModule, NotificationService } from "./notification.module";
-import { I18n } from "../i18n/i18n.module";
 import { DocumentationModule } from "./../documentation-component/documentation.module";
+import { Subject } from "rxjs";
+
+@Component({
+	selector: "app-notification-action-story",
+	template: `
+		<ibm-notification [notificationObj]="{
+			type: 'error',
+			title: 'Sample notification',
+			message: 'Sample error message',
+			showClose: showClose,
+			lowContrast: lowContrast,
+			actions: actions}">
+		</ibm-notification>
+		<ibm-notification [notificationObj]="{
+			type: 'info',
+			title: 'Sample notification',
+			message: 'Sample error message',
+			showClose: showClose,
+			lowContrast: lowContrast,
+			actions: actions}">
+		</ibm-notification>
+		<ibm-notification [notificationObj]="{
+			type: 'success',
+			title: 'Sample notification',
+			message: 'Sample error message',
+			showClose: showClose,
+			lowContrast: lowContrast,
+			actions: actions}">
+		</ibm-notification>
+		<ibm-notification [notificationObj]="{
+			type: 'warning',
+			title: 'Sample notification',
+			message: 'Sample error message',
+			showClose: showClose,
+			lowContrast: lowContrast,
+			actions: actions}">
+		</ibm-notification>
+		`,
+	providers: [NotificationService]
+})
+class NotificationActionStory {
+	@Input() showClose = true;
+	@Input() lowContrast = false;
+
+	@Output() actionClicked: EventEmitter<any> = new EventEmitter();
+
+	actionSubject = new Subject<any>();
+
+	actions = [
+		{
+			text: "Action",
+			click: this.actionSubject.subscribe({
```

Comment:
The `isObservable` check isn't working because you're not passing an observable to the component - you're passing a subscription (`.subscribe` returns a `Subscription`) ... Maybe add a `ngOnInit` and subscribe to the action there 👍

Reply:
It still seems to be type `object` when I pass in only actionSubject without subscribing.

Reply:
Odd ... what happens if you initialize `actions` in `ngOnInit`?